### PR TITLE
Add some butter instance cleanup to the test, fix bug in waitForMedia ch...

### PR DESCRIPTION
**Pull request by humphd**
...eck with _popcorn.destroy()

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brettgaylor received. Current queue size: 0

Live output at: http://botio.butyr.org:80/ba044702e0a1429/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:80/ba044702e0a1429/output.txt

Total script time: 0.87 mins

Server is running on http://pr318.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:80/9cb2ac81c427b25/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:80/9cb2ac81c427b25/output.txt

Total script time: 0.15 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:80/bd0519b07e7fa10/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:80/bd0519b07e7fa10/output.txt

Total script time: 0.42 mins

Server is running on http://pr318.butyr.org/templates/test.html

File: src/core/popcorn-wrapper.js [hunk 1]
```diff
@@ -403,7 +403,11 @@ define( [ "core/logger", "core/eventmanager" ], function( Logger, EventManager )
      */
     function waitForMedia( readyCallback, timeoutCallback ){
       checkTimeoutLoop(function(){
-        return ( _popcorn.media.readyState >= 1 && _popcorn.duration() > 0 );
+        return ( _popcorn && /* Make sure _popcorn still exists (e.g., destory() hasn't been called) */
```
Comment (cadecairos) [inline src/core/popcorn-wrapper.js]: destory -> destroy

**Pull request merged**